tutorials/replicating-large-datasets/README.md

```diff
@@ -0,0 +1,139 @@
+	- [Step 5: Pin the Data on Your node](#step-5-pin-the-data-on-your-node)
+
+<!-- /TOC -->
+
+# Background
+
+These instructions apply to either of these cases:
+* You are replicating data between a set of known peers
+* You want to set up a _seed node_ with a public IP address that everyone can pull the initial copies of the data from.
+
+For example, after Jack [downloads all of the data.gov datasets](https://github.com/ipfs/archives/issues/113), he needs a way to let his collaborators replicate the datasets efficiently. Eventually they will all be able to serve the data on the general, distributed network of IPFS peers, but the first priority is to pass complete replicas to the collaborators who've allocated enough storage to hold the entire dataset.
+
+In order to do this you're going to turn off the DHT and peer auto-discovery features. These features are important, amazing parts of IPFS but they slow down the system, eat up a bunch of bandwidth, and they're not needed in a situation where you already know which nodes you'll be communicating with.
+
+# Instructions for Providers/Sources
+
+If you have datasets on your machine and want to serve them over the network, follow these instructions. They will walk you through using ipfs-pack to register and serve your data.
+
+## TL;DR.
+
+If you just want to run the commands without explanation, here's what you need to do. _This assumes that you've already installed [ipfs-pack](https://github.com/ipfs/ipfs-pack)._
+
+It's best if you do this on a machine that has a public IP address.
+
+```
+cd /path-to-your/dataset-directory
+ipfs-pack make
+ipfs-pack serve
+```
+
+And then retrieve the Pack Root and the multiaddr for the node you just started. This info was printed out on the console when you ran `ipfs pack serve`. Give the multiaddr and Pack Root to the people who are setting up Mirrors.
+
+## Step 1: Use ipfs-pack to serve your datasets
+
+Follow the instructions in [the ipfs-pack tutorial](https://github.com/ipfs/ipfs-pack/blob/master/tutorial/README.md), which covers installing ipfs-pack, initializing a pack, and serving the contents of your pack on the IPFS network.
+
+## Step 2: Get the multiaddr of your node
+
+This works best if you're running ipfs on a machine with a public IP address.
+
+After starting the ipfs node with `ipfs-pack serve`, you will see some info about the node printed on the console. It will look like:
+
+```
+verified pack, starting server...
+Serving data in this pack...
+Peer ID: QmVbXV7mQ5Fs3tYY2Euek5YdkkzcRafUg8qGWvFdgaBMuo
+    /ip4/127.0.0.1/tcp/58162
+    /ip4/1.2.3.4/tcp/58162
+Pack root is QmRguPt6jHmVMzu1NM8wQmpoymM9UeqDJGXdQyU3GhiPy4
+Shared:      0 blocks, 0 B total data uploaded
+```
+
+The multiaddr is the public IPv4 address plus the Pack Peer ID, so for the sample output above, your pack's multiaddr would be `/ip4/1.2.3.4/tcp/58162/QmVbXV7mQ5Fs3tYY2Euek5YdkkzcRafUg8qGWvFdgaBMuo`
```

Comment:
Should be: ``/ip4/1.2.3.4/tcp/58162/ipfs/QmVbXV7mQ5Fs3tYY2Euek5YdkkzcRafUg8qGWvFdgaBMuo` (missing the `/ipfs/` bit before the peer ID)